src/payment/checkout/Checkout.jsx

```diff
@@ -111,7 +111,7 @@ class Checkout extends React.Component {
             />
           </h5>
 
-          <p className="d-flex flex-wrap">
+          <p data-hj-suppress className="d-flex flex-wrap">
```

Comment:
I don't think we need to hide the buttons.

Reply:
@waheedahmed Yes putting on parent would suppress its child's as well.
Regarding the unsuppressed you have mentioned, My understanding is these fields have keystroke data it would be automatically suppressed. 
I have added the suppression for the fields which have select option e.g country, year, month